src/cpp/multisolver_interface/SolverCbc.cpp

```diff
@@ -251,6 +251,11 @@ void SolverCbc::read_basis(const std::filesystem::path& filename)
     defineCbcModelFromInnerSolver();
 }
 
+void SolverCbc::set_basis(std::span<int> rstatus, std::span<int> cstatus)
+{
+    _cbc.solver()->setBasisStatus(rstatus.data(), cstatus.data());
```

Comment:
Sure about this ? Looking at the previous function, I would have tested `_clp_inner_solver.getModelPtr()->setBasisStatus` or even manually `setRowStatus` and `setColStatus` as in clp